### PR TITLE
added passenger_boarding and passengar_alighting fields to stop_time table

### PR DESCRIPTION
**Pull request by peetjvv**
- backwards compatible
 - also fixed the missing agency_email field from postgres db's

### CODE REVIEW

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLStopTimeCollection.cs [hunk 1]
```diff
@@ -263,7 +277,9 @@ public IEnumerable<StopTime> Get()
                         StopHeadsign = reader.ReadStringSafe(),
                         PickupType = (PickupType?)reader.ReadIntSafe(),
                         DropOffType = (DropOffType?)reader.ReadIntSafe(),
-                        ShapeDistTravelled = reader.ReadStringSafe()
+                        ShapeDistTravelled = reader.ReadStringSafe(),
+                        PassengerBoarding = (uint)reader.Read<int>(NpgsqlTypes.NpgsqlDbType.Integer),
```
Comment (timminata) [inline GTFS.DB.PostgreSQL/Collections/PostgreSQLStopTimeCollection.cs]: Why aren't you using read int safe? What if these are null? Surely in most cases they will be null? Have you tested/converted any db's yet?

File: GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs [hunk 2]
```diff
@@ -249,7 +249,11 @@ private void RebuildDB()
             // Alter existing tables, if their structure is incorrect, for backwards compatibility
             //  1. add agency_email column to agency
             if (!ColumnExists("agency", "agency_email")) this.ExecuteNonQuery("ALTER TABLE [agency] ADD [agency_email] TEXT;");
-            
+            //  2. add passenger_boarding column to stop_time
+            if (!ColumnExists("stop_time", "passenger_boarding")) this.ExecuteNonQuery("ALTER TABLE [stop_time] ADD [passenger_boarding] INTEGER;");
```
Comment (timminata) [inline GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs:253]: If I'm not mistaken all values will be null when the column is added, which is fine, as long as it's being handled

Comment (peetjvv) [reply GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs:253]: They're all null, yes. I haven't done a test to see if the wizard will open an existing route. I'll do one now.

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLStopTimeCollection.cs (hunk 1 above)
Comment (peetjvv) [reply GTFS.DB.PostgreSQL/Collections/PostgreSQLStopTimeCollection.cs]: done. no way to test it until we import a feed with boarding and alighting to postgres though

**Pull request merged**